packages/graph/hash_graph/lib/graph/src/store/postgres/pool.rs

```diff
@@ -46,7 +46,7 @@ where
     ///
     /// - if creating a connection returns an error.
     pub async fn new(db_info: &DatabaseConnectionInfo, tls: Tls) -> Result<Self, StoreError> {
-        tracing::debug!("Creating connection pool to Postgres");
+        tracing::debug!(url=%db_info, "Creating connection pool to Postgres");
```

Comment:
I can't see the struct but does this print passwords or anything?

Reply:
No, password will be masked with `***`